homeassistant/components/reolink/update.py

```diff
@@ -0,0 +1,72 @@
+"""Update entities for Reolink devices."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from homeassistant.components.update import (
+    UpdateDeviceClass,
+    UpdateEntity,
+    UpdateEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import ReolinkData
+from .const import DOMAIN
+from .entity import ReolinkBaseCoordinatorEntity
+
+LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up update entities for Reolink component."""
+    reolink_data: ReolinkData = hass.data[DOMAIN][config_entry.entry_id]
+    async_add_entities([ReolinkUpdateEntity(reolink_data)])
+
+
+class ReolinkUpdateEntity(ReolinkBaseCoordinatorEntity, UpdateEntity):
+    """Update entity for a Netgear device."""
+
+    _attr_has_entity_name = True
```

Comment:
You could move `_attr_has_entity_name` to base class I would think

Reply:
Good idea, just done it in the latest commit, thanks!